litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -563,6 +563,31 @@ spec:
             [
               "while [[ $(curl -sw '%{http_code}' http://mongo-service:27017 -o /dev/null) -ne 200 ]]; do sleep 5; echo 'Waiting for the MongoDB to be ready...'; done; echo 'Connection with MongoDB established'",
             ]
+        - name: server-version-updater
+          image: litmuschaos/mongo-utils:latest
+          command: ["/bin/sh"]
+          args: ["-c"," mongo  $DB_SERVER --username $DB_USER --password $DB_PASSWORD --eval 'db=db.getSiblingDB(\"litmus\");db.getCollection(\"server-config-collection\").updateOne({\"key\":\"version\"},{ \"$set\":{ \"value\":\"'$VERSION'\"}})'"]
+          env:
+            - name: VERSION
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: VERSION
+            - name: DB_USER
+              valueFrom:
+                secretKeyRef:
+                  name: litmus-portal-admin-secret
+                  key: DB_USER
+            - name: DB_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  name: litmus-portal-admin-secret
+                  key: DB_PASSWORD
+            - name: DB_SERVER
+              valueFrom:
+                configMapKeyRef:
+                  name: litmus-portal-admin-config
+                  key: DB_SERVER   
```

Comment:
can be updated to

```envFrom:
    - configMapRef:
           name: litmus-portal-admin-config

Reply:
@Jonsy13

Reply:
We are taking values from secret as well as configMap.

Reply:
then we can add
```
- secretRef:
            name: litmus-portal-admin-secret
```

Reply:
@Jonsy13 

```
envFrom:
    - configMapRef:
           name: litmus-portal-admin-config
    - secretRef:
           name: litmus-portal-admin-secret
```

Reply:
@rajdas98 Do we need all secrets to be passed to initContainer?